ckanext/dalrrd_emc_dcpr/templates/data_capture.html

```diff
@@ -1,19 +1,21 @@
 {% extends "page.html" %}
-{% block subtitle %}{{ _("Welcome DALRRD") }}{% endblock %}
+{% block subtitle %}{{ _("Data Capture Project Register (DCPR)") }}{% endblock %}
 {% block content %}
-    <section class="bg-light py-5 border-bottom" xmlns="http://www.w3.org/1999/html">
-            <div class="container-fluid px-5 my-5">
-                <div class="mb-5">
+    <div class="container">
+    <section class="py-5 border-bottom" xmlns="http://www.w3.org/1999/html">
+                <div class="mb-3">
                     <h1 class="fw-bolder mb-5">Data Capture Project Register (DCPR)</h1>
                    <span class="text-muted">
                         In terms of section 6.1.3(d) of the Base Dataset Custodianship Policy, base dataset custodians must register their </br>
                        data capture projects on the Data Capture Project Register (DCPR) prior to undertaking projects and update the register when
                        </br>changing the status of projects.
                    </span>
                     <h5 class="mb-0 fw-bolder mb-5">The purpose DCPR is to achieve the following:</h5>
-                    <p>Reduce duplication of spatial data capture</p>
-                    <p>Ensure efficient, effective and economical spatial data capture</p>
-                    <p>Facilitate the sharing of spatial information</p>
+                    <ul class="text-muted">
+                    <li>Reduce duplication of spatial data capture</li>
+                    <li>Ensure efficient, effective and economical spatial data capture</li>
+                    <li>Facilitate the sharing of spatial information</li>
+                    </ul>
 
```

Comment:
@Fanevanjanahary please add some screenshots with UI changes css/html, although this particular change is obvious, usually it's not.

Reply:
Hi @Mohab25 
![image](https://user-images.githubusercontent.com/12623986/212620187-f6ec4315-8380-47d7-9d3c-231e767acbb8.png)